api/krusty/remotelocalizer_test.go

```diff
@@ -0,0 +1,542 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package krusty_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	loc "sigs.k8s.io/kustomize/api/internal/localizer"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+const simpleHTTPS = "https://github.com/kubernetes-sigs/kustomize//api/krusty/testdata/localize/simple"
```

Comment:
Since #4783 has been merged, I believe I could replace these urls with local, `file://`-prefixed urls? I'd have to look into it, but lmk. If I made this change, I'd add tests for [`parseDomain`](https://github.com/kubernetes-sigs/kustomize/pull/4797/files#diff-f52a1bf07c678854203647b33ccaa09d945af03c9233b3dec3a947ba27d531deR154).